src/main/java/seedu/address/model/person/lab/LabStatus.java

```diff
@@ -1,5 +1,43 @@
 package seedu.address.model.person.lab;
 
+import seedu.address.logic.parser.exceptions.ParseException;
+
 public enum LabStatus {
     UNSUBMITTED, SUBMITTED, GRADED;
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "LabStatus should be u, s, or g, which respectively stands for "
+            + "Unsubmitted, Submitted, and Graded";
+
+    public static final String VALIDATION_REGEX = "[usg]";
+
+    /**
+     * Returns true if a given string is a valid LabStatus
+     * @param test
+     */
+    public static boolean isValidLabStatus(String test) {
+        return test.matches(VALIDATION_REGEX);
+    }
+
+    /**
+     * Maps the string given into LabStatus
+     * @param toMap String to be mapped
+     * @return {@code LabStatus} corresponding to the String given
+     * @throws ParseException when invalid LabStatus is given
+     */
+    public static LabStatus map(String toMap) throws ParseException {
```

Comment:
Perhaps parsing the input to its corresponding `LabStatus` should be handled by `ParserUtil` rather than this class?

Reply:
Ah yes, I agree with @channne. Good suggestion.

Reply:
You're right. I will move the implementation into <code>ParserUtil</code>